botbuilder-teams-js/src/activityProcessor/teamsInvoke.ts

```diff
@@ -214,4 +220,26 @@ export class InvokeActivity {
       }
     }
   }
+
+  private static striHtmlTag(content: string): string {
+    let dom = new JSDOM(content);
+    const textRestrictedHtmlTags = new Set(["AT", "ATTACHMENT"]);
+    return this.striHtmlTagHelper(dom.window.document.body, textRestrictedHtmlTags);
+  }
+
+  private static striHtmlTagHelper(node, tags: Set<string>): string {
+    let result = '';
+    if (tags.has(node.tagName)) {
+      result += node.outerHTML;
+    } else {
+      node.childNodes.forEach(childNode => {
+        if (childNode.nodeType === 3) {
```

Comment:
I believe if you install `@types/jsdom` then you'll have constant or enum values here instead of raw "3"

Reply:
fixed